plugin/proto/msg.proto

```diff
@@ -45,8 +45,11 @@ message EthereumConfig {
   bytes eip150BlockNumber = 3;
   bytes eip160BlockNumber = 4;
   bytes eip161BlockNumber = 5;
-  bytes maxCodeSize = 6;
```

Comment:
Please don't renumber the fields unnecessarily. Just add the new ones at the end.